### ISSUE TITLE
PF3 Dropdown Kebab accessibility enhancements

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#762]
When using the [dropdown kebab](https://395-pr-patternfly-react-patternfly.surge.sh/patternfly-3/index.html?selectedKind=patternfly-react%2FWidgets%2FDropdown%20Kebab&selectedStory=DropdownKebab&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs) in another component, the aXe tooling will flag a11y violations which cannot be addressed using the current kebab implementation.

For example, refer to this [comment in the Dual List Selector](https://github.com/patternfly/patternfly-react/pull/395#issuecomment-429609165).

When trying to add the missing attributes mentioned in the errors (`aria-label` and its friends),
the component doesn't seem to accept them and add them to the actual DOM element.

For this issue, update the Dropdown Kebab so that these accessibility issues are either addressed by the component or can be addressed when using the component.

Note that some aria attributes should be customizable by the consumer, like `aria-label`.
